### DISCUSSION TITLE
range slider with two inputs

### DISCUSSION THREAD

**Discussion by hodanli** [saadeghi/daisyui#1136] [Q&A] [answered]
hi,
how can i make slider with two inputs like in the image below?

![image](https://user-images.githubusercontent.com/6535865/190164751-da8fcab7-f247-4346-a440-92573655e5c4.png)

**Comment by saadeghi** [comment, marked as answer]
See https://github.com/saadeghi/daisyui/issues/573#issuecomment-1064453159
